### PR TITLE
Update German translation

### PR DESCRIPTION
**Pull request by marcelgerber**
Kind of wonky this time around.
@ingorichter @pthiess

### CODE REVIEW

File: src/nls/de/strings.js [hunk 1]
```diff
@@ -767,8 +774,12 @@ define({
...
+    "DESCRIPTION_SHOW_PANE_HEADER_BUTTONS"           : "Zeigt bei der geteilten Ansicht die Schließen- und Umschalten-Buttons in der Kopfzeile",
+
     "DEFAULT_PREFERENCES_JSON_HEADER_COMMENT"        : "/*\n * Dies ist ein schreibgeschütztes Dokument, das alle von\n * {APP_NAME} unterstützten Einstellungen auflistet.\n * Nutzen Sie dieses Dokument als Referenz, um die\n * Einstellungsdatei \"brackets.json\", die in der anderen\n * Ansicht geöffnet ist, anzupassen.\n * Besuchen Sie auch https://github.com/adobe/brackets/wiki/How-to-Use-Brackets#preferences,\n * um mehr über den Umgang mit Einstellungen in {APP_NAME}\n * zu erfahren.\n */",
-    "DEFAULT_PREFERENCES_JSON_DEFAULT"               : "Standard"
+    "DEFAULT_PREFERENCES_JSON_DEFAULT"               : "Standard",
+    "DESCRIPTION_PURE_CODING_SURFACE"                : "Aktiviert den Ablenkungsfreien Modus, bei dem alle UI-Elemente außer dem Code ausgeblendet werden"
 });
 
 /* Last translated for 1eba0ae1cb21e73752c8359132b972ec7e5997ab */
```
Comment (petetnt) [inline src/nls/de/strings.js]: SHA-1 needs to be updated here :memo:

Comment (petetnt) [inline src/nls/de/strings.js]: SHA-1 needs to be updated here :memo:

Comment (MarcelGerber) [inline src/nls/de/strings.js]: Solid point, thanks. Updated already :)

**Comment by abose** [comment]
@MarcelGerber missed this for 1.6 :sweat_smile: 
Can this be retargetted to master?

**Comment by abose** [comment]
Merging for build refresh of 1.6

**Pull request merged**